packages/components/app/styles/components/accordion.scss

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// ACCORDION
+//
+
+@use "../mixins/button" as *;
+@use "../mixins/focus-ring" as *;
+
+$accordion-item-padding: 16px;
+$accordion-item-medium-border-radius: 6px;
```

Comment:
I don't understand what `medium` means in this context.

Reply:
I called it "medium" because that's how that size (6px) is referred to in the documentation here: https://helios.hashicorp.design/foundations/border

I'll remove it though